website/docs/components/table/advanced-table/partials/accessibility/accessibility.md

```diff
@@ -14,6 +14,78 @@ The table row element cannot receive interactions, meaning actions cannot be att
 
 Unlike the Table component, each cell receives focus in the Advanced Table to support users navigating through the table efficiently with a keyboard. For any other interactions, you must use interactive elements (buttons, links, etc.) within the cells. 
 
+## Keyboard Navigation
+
+The Advanced Table supports two keyboard interaction modes: Navigation Mode and Action Mode. These modes allow users to move through table data and interact with elements using a keyboard.
+
+### Navigation Mode
+
+
+<Doc::Badge @type="neutral">Tab</Doc::Badge>
+
+When a user tabs into the Advanced Table, the first cell is focused.
+
+![Keyboard focus on the sortable 'Artist' column header with sorting controls active.](/assets/components/table/advanced-table/advanced-table-keyboard-navigation-tab.png =402x*)
+
+<Doc::Badge @type="neutral"><Hds::Icon @name="arrow-left" /></Doc::Badge> <Doc::Badge @type="neutral"><Hds::Icon @name="arrow-right" /></Doc::Badge><Doc::Badge @type="neutral"><Hds::Icon @name="arrow-up" /></Doc::Badge> <Doc::Badge @type="neutral"><Hds::Icon @name="arrow-down" /></Doc::Badge>
```

Comment:
Need to add a title to the icons because there is no visible text associated with them.

```suggestion
<Doc::Badge @type="neutral"><Hds::Icon @name="arrow-left" @title="Left arrowy" /></Doc::Badge> <Doc::Badge @type="neutral"><Hds::Icon @name="arrow-right" @title="Right arrow" /></Doc::Badge><Doc::Badge @type="neutral"><Hds::Icon @name="arrow-up" @title="Up arrow" /></Doc::Badge> <Doc::Badge @type="neutral"><Hds::Icon @name="arrow-down" @title="Down arrow" /></Doc::Badge>
```